templates/index.html

```diff
@@ -22,8 +22,7 @@
 #}
 
 {% block takeover_content %}
-  {% include "takeovers/_german_takeover_k8.html" %}
-  {% include "takeovers/_german_takeover_openstack_made_easy.html" %}
```

Comment:
Don't delete the german ones please

Reply:
@anthonydillon added the changes. Cheers